lib/cdo/aws/cloud_formation.rb

```diff
@@ -220,6 +222,114 @@ def create_or_update
         end
       end
 
+      def start_inactive_instance
+        cloudformation_resource = Aws::CloudFormation::Resource.new
+        stack = cloudformation_resource.stack(stack_name)
+        instance = Aws::EC2::Instance.new(id: stack.resource('WebServer').physical_resource_id)
+        if instance.state.code != 80
+          CDO.log.info "Instance #{instance.id} in Stack #{stack_name} can't be started because it is not" \
+          " currently stopped.  Current state - #{instance.state.code}:#{instance.state.name}"
+        else
+          CDO.log.info "Starting Instance #{instance.id} ..."
+          instance.start
+          instance.wait_until_running
+          CDO.log.info "Instance #{instance.id} is started."
+
+          public_ip_address = instance.reload.public_ip_address
+          dashboard_url = stack.outputs.select {|output| output.output_key == 'DashboardURL'}.first.output_value
+          pegasus_url = stack.outputs.select {|output| output.output_key == 'PegasusURL'}.first.output_value
```

Comment:
`#select {[...]}.first` can be simplified to [`#detect`](https://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-detect)` {[...]}`

Reply:
Done!